registry/src/pages/login.js

```diff
@@ -84,6 +86,12 @@ const Login = () => {
         </p>
       </form>
     </Container>
+   ) : (
```

Comment:
Just make the text inside "Login Button" change to login indicator instead of changing the whole page.

Reply:
sure.